package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "playwright-core",
-  "version": "0.11.1-post",
+  "version": "0.12.0",
```

Comment:
what about other packages, should we bump their versions too?

Reply:
Good point!